src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -425,13 +440,15 @@ public void simulationPeriodic() {
     }
 
     // Wrist Sim Stuff
-    m_wristMotorSim.setInput(wristMotor.get() * RobotController.getBatteryVoltage());
+    m_wristMotorSim.setInput(wristMotor.getAppliedOutput());
     m_wristMotorSim.update(Robot.defaultPeriodSecs);
     simWristPos.set(Units.radiansToRotations(m_wristMotorSim.getAngleRads()));
+    simWristVel.set(Units.radiansToRotations(m_wristMotorSim.getVelocityRadPerSec()));
     // Offest added so that gravity is simulated in the right direction
     wristMech.setAngle(Units.radiansToDegrees(m_wristMotorSim.getAngleRads()) - 90);
+    SmartDashboard.putNumber("Current Draw Wrist (A)", m_wristMotorSim.getCurrentDrawAmps());
 
-    SmartDashboard.putNumber("Wrist Motor Sim Output", wristMotor.get());
+    SmartDashboard.putNumber("Wrist Motor Sim Output", wristMotor.getAppliedOutput());
```

Comment:
This would be good to log in periodic, not just in simulation.

Reply:
Fixed in the Testing branch